rasa/nlu/utils/pattern_utils.py

```diff
@@ -49,7 +53,9 @@ def _generate_lookup_regex(lookup_table: Dict[Text, Union[Text, List[Text]]]) ->
 
     # sanitize the regex, escape special characters
     elements_sanitized = [re.escape(e) for e in elements_to_regex]
-
+    not_boundary_language = ["zh", "ja", "th"]
```

Comment:
can we move these into a constant `NON_WHITESPACE_LANGUAGES` in `rasa/nlu/constants.py`?

Reply:
Of course !